docs/_docs/plugins/installation.md

```diff
@@ -3,77 +3,123 @@ title: Plugins
 permalink: /docs/plugins/installation/
 ---
 
-You have 3 options for installing plugins:
+Jekyll has built-in support for using plugins to extend the core functionality.
 
-1. In your site source root, make a `_plugins` directory. Place your plugins
-   here. Any file ending in `*.rb` inside this directory will be loaded before
-   Jekyll generates your site.
+Primarily, any file with extension `.rb` placed within a `_plugins` directory at the root of the site's `source`, will be automatically loaded
+during a build session.
 
-2. In your `_config.yml` file, add a new array with the key `plugins` (or `gems` for Jekyll < `3.5.0`) and the
-   values of the gem names of the plugins you'd like to use. An example:
+This behavior can be configured as per following:
 
-   ```yaml
-   # This will require each of these plugins automatically.
-   plugins:
-     - jekyll-gist
-     - jekyll-coffeescript
-     - jekyll-assets
-     - another-jekyll-plugin
-   ```
+  * The `_plugins` directory may be changed either directly via the command-line or via the configuration file(s).
+  * Plugins in the `_plugins` directory (or its equivalent(s)) will not be loaded during the `safe` mode.
+  * This route cannot be used to extend the Jekyll CLI.
 
-   Then install your plugins using `gem install jekyll-gist jekyll-coffeescript jekyll-assets another-jekyll-plugin`
+To work with plugins packaged as gems, one has to list the desired gems in the configuration file under a top-level key named `plugins`
```

Comment:
What do you think about removing the documentation for the `gems` key? I don't think it makes sense to document behavior for anything older than Jekyll 3.9.x.